card/mod/pointer/set/abstract/02_pointer/html_views.rb

```diff
@@ -104,7 +104,7 @@ def add_item_modal_link
                        filter_card: filter_card.name,
                        slot_selector: filtered_list_slot_class,
                        item_selector: "_filtered-list-item",
-                       slot: { hide: :modal_footer },
+                       slot: { hide: [:modal_footer] },
```

Comment:
Is this necessary? If so then it should change.

Reply:
If you're asking whether hiding the modal footer necessary, I don't know the answer to that.
If you're asking whether putting the `hide` values in an array is necessary, then I think the answer is that a simple string (or symbol) will work if it's the only thing hidden, but in this case there was _another_ `hide` configuration for that slot from somewhere else.  So the params looked something like `slot[hide]=A&slot[hide][]=...`.

The problem was showing up in editing company and metric lists on wikirate project pages. This was a quick fix that got it working (it's already deployed).